packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -451,6 +451,7 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
                                     <MobileDrawer.Item
                                         onClick={() => {
                                             toggleDrawer();
+                                            setShouldShowGlobalLoader(true);
```

Comment:
Why do we need loader here @sergei-deriv 🙏

Reply:
@mitra-deriv We have an issue, when the user trying to logout from account switcher on the route which is behind the auth (/cashier or /account, for example), if we just call logout method from client-store, in this case user will be redirected to login page instead of deriv.com, because application make logout and in this time we are checking this route is behind the route and redirect the user to login page and we don't have time to redirect him to deriv.com. To solve this problem we need to redirect the user from route which is behind the auth and make logout, but in this case the user will see logged in version of TH, then he will see logged out version of TH and only after this he will be redirected to deriv.com... To hide this flow from the user I've added this loader

Reply:
So we need the loader for all these 3 places to fix that :( hopefully will be fixed by just showing them loggedout version instead of deriv.com , thanks for the explanation 🙏